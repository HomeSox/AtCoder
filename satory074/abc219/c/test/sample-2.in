zyxwvutsrqponmlkjihgfedcba
10
a
ab
hanada
abc
ac
unbobo
b
z
zunbobo
zundoko