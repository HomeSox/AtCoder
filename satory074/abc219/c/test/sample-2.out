z
zundoko
zunbobo
unbobo
hanada
b
a
ac
ab
abc
